client/src/components/inventory/inventoryHeading.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import './inventory.scss';
+
+class inventoryHeading extends Component {
+    render() {
+        return (
+            <div>
+                <h2>Inventory</h2>
```

Comment:
should be h1, then h1 mobile in css file

Reply:
Yeah I'm not even close to being done. Im just trying to pull develop but it wont let me unless I pushed my changes